spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/ComposedTaskProperties.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.config.features;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Glenn Renfro
+ */
+@ConfigurationProperties(prefix = ComposedTaskProperties.COMPOSED_TASK_PREFIX)
+public class ComposedTaskProperties {
+
+
+	public ComposedTaskProperties() {
+		try {
+			composedTaskRunnerUri = new URI("maven://org.springframework.cloud.task.app:composedtaskrunner-task:1.0.0.BUILD-SNAPSHOT");
+		}
+		catch (URISyntaxException e) {
+			throw new IllegalStateException(
+					"Invalid URI specified for composedTaskRunnerUri");
+		}
+	}
+
+	public static final String COMPOSED_TASK_PREFIX = "spring.cloud.dataflow.composed.task";
+
+	private URI composedTaskRunnerUri;
+
+
+	private String taskName = "composed-task-runner";
+
+	public URI getComposedTaskRunnerUri() {
+		return composedTaskRunnerUri;
+	}
+
+	public void setComposedTaskRunnerUri(String composedTaskRunnerUri) {
```

Comment:
This setter should accept a URI, that was the whole point of my previous comment: let boot binding do the conversion. If this is not possible, then I'd argue this brings little value (at the very least, should throw IllegalArgumentException instead)